exercises/bowling/canonical-data.json

```diff
@@ -98,6 +102,10 @@
       "rolls": [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 10, 6, 10],
       "expected": -1
     }, {
+      "description": "second bonus roll after a strike in the last frame can not score than 10 points",
+      "rolls": [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 10, 10, 11],
```

Comment:
unsure about the necessity of this one. But maybe it's useful - some people may only check the first bonus roll?